product_docs/docs/pgd/5.6/overview/architecture-and-performance.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: "PGD overview - architecture and performance"
 navTitle: Architecture and performance
-description: EDB Postgres Distributed architectural options and performance characteristics, including always-on architectures, supported Postgres database servers, and characteristics affecting performance
+description: EDB Postgres Distributed architectural options and performance characteristics, including Always-on architectures, supported Postgres database servers, and characteristics affecting performance.
 deepToC: true
 redirects: 
   - bdr
```

Comment:
@djw-m line 22, is the word "and" missing? Or is there something else missing here? I couldn't follow the logic of this sentence otherwise. Rest of the file looked great!

If, at any point, the write leader is seen to be unavailable, the remaining nodes in the group select a new write leader from the group **and** the proxies direct traffic to that node.